TestownikConsoleApp/WriteLinesClass.cs

```diff
@@ -56,7 +56,7 @@ public void WriteLines(string path)
 
             foreach (var y in selectedAnswers)
             {
-                if (result.Contains(y.Type))
+                if (result.Contains(y.Type)) //Robi tyle razy ile jest odpowiedzi przez foreacha, jak to ograniczyc do jednego razu
```

Comment:
Ogólnie słabe to sprawdzenie. Masz tam obiekt 'split'. Spróbuj poiterować po nim wcześniej i sprawdzić czy w 'typeOfAnswers' jest taka litera, jeżeli nie to wyrzucasz błąd 'Zły znak, wprowadź odpowiedź!'. A jeżeli wszystkie są to pozwalasz mu dalej wykonać kod i później już tego nie sprawdzasz